helm/mds/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "0.4.0"
+appVersion: "0.0.1"
```

Comment:
Why is the version being downgraded?

Reply:
Merge was a bit messy initially (showing changed files that weren't changed) so just replaced this dir, seeing now that I need to go back and work out merge properly.